components/Downloads/DownloadButton.tsx

```diff
@@ -0,0 +1,50 @@
+'use client';
+
+import { ArrowRightIcon } from '@heroicons/react/24/outline';
+import type { FC, PropsWithChildren } from 'react';
+
+import Button from '@/components/Common/Button';
+import { useDetectOS } from '@/hooks';
+import { useRouter } from '@/navigation.mjs';
+import type { NodeRelease } from '@/types';
+import { downloadUrlByOS } from '@/util/downloadUrlByOS';
+
+type DownloadButtonProps = { release: NodeRelease };
+
+const DownloadButton: FC<PropsWithChildren<DownloadButtonProps>> = ({
+  release: { versionWithPrefix },
+  children,
+}) => {
+  const { os, bitness } = useDetectOS();
+  const { push } = useRouter();
+
+  const nodeDownloadLink = downloadUrlByOS(versionWithPrefix, os, bitness);
+
+  const onDownloadNode = () => push(nodeDownloadLink);
+
+  return (
+    <>
+      <Button
+        variant="special"
+        onClick={onDownloadNode}
+        className="hidden flex-row items-center justify-center gap-2 dark:flex"
```

Comment:
Curious why these tailwind classes (and the ones below) are inline? I thought we were trying to use CSS modules

Reply:
True. I was experimenting here, but these should def go to CSS modules